examples/timelines/convert.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
```

Comment:
I would add in a comment a sample invocation as a simple way to document how to use the script.

Reply:
Spoke too soon. There are not options for the script it's all in the same dir. Neverming